### PR TITLE
Creating releasing document

### PR DESCRIPTION
**Pull request by Albertoimpl**

### CODE REVIEW

File: RELEASING.adoc [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+2022-01-14 11:39:48.195  INFO 21 --- [           main] i.s.c.releasescripts.Application         : No active profile set, falling back to default profiles: default
+2022-01-14 11:39:49.399  INFO 21 --- [           main] i.s.c.releasescripts.Application         : Started Application in 2.012 seconds (JVM running for 2.747)
+2022-01-14 11:39:49.401 DEBUG 21 --- [           main] i.s.c.r.command.CommandProcessor         : Running command processor
+2022-01-14 11:39:49.402 DEBUG 21 --- [           main] i.s.c.r.command.CommandProcessor         : Found command io.spring.concourse.releasescripts.command.PromoteCommand
+2022-01-14 11:39:49.402 DEBUG 21 --- [           main] i.s.c.r.command.PromoteCommand           : Running 'promote' command
+2022-01-14 11:39:49.495  INFO 21 --- [           main] i.s.c.r.artifactory.ArtifactoryService   : Promoting spring-cloud-app-broker/app-broker-1.4.x-stage-release-1 to libs-release-local
+2022-01-14 11:39:55.938 DEBUG 21 --- [           main] i.s.c.r.artifactory.ArtifactoryService   : Promotion complete
+Promotion complete
+----
+
```
Comment (royclarkson) [inline RELEASING.adoc:136]: This is where I merge the release tag into main or the branch so as to maintain a linear history.

File: RELEASING.adoc [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+2022-01-14 11:46:03.699  INFO 17 --- [           main] i.s.c.r.sonatype.SonatypeService         : Deploy complete. Closing staging repository
+2022-01-14 11:46:03.803  INFO 17 --- [           main] i.s.c.r.sonatype.SonatypeService         : Close requested. Awaiting result
+2022-01-14 11:46:49.673  INFO 17 --- [           main] i.s.c.r.sonatype.SonatypeService         : Staging repository closed
+2022-01-14 11:46:49.763  INFO 17 --- [           main] i.s.c.r.sonatype.SonatypeService         : Staging repository released
+Sync complete
+----
+
+After that step succeeded, we can go and create a new release in GitHub adding the newly created tag and writing the release notes: https://github.com/spring-cloud/spring-cloud-app-broker/releases
+
+The version bump and the branching does not happen automatically anymore, so we need to do the following:
```
Comment (royclarkson) [inline RELEASING.adoc:322]: I thought the release script bumped the version on the release tag that gets created? But you then must merge the tag back in manually.

File: RELEASING.adoc (hunk 1 above)
Comment (Albertoimpl) [reply RELEASING.adoc:136]: Ah, I see what I did differently this time. I will merge that manually and update this document. Thanks!

Comment (royclarkson) [reply RELEASING.adoc:136]: There's an argument to be made that you shouldn't do this until it's published to maven central for GA releases. We have the ability to back out artifacts from artifactory, so we could try to reset if something fails on maven central.

**Review by royclarkson — APPROVED** [review]
👍

**Pull request merged**